sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Breaking Changes
 
 - Removed `Azure::Storage::Blobs::PackageVersion`.
+- Removed `PageBlobClient::GetPageRanges`, `PageBlobClient::GetPageRangesDiff` and `PageBlobClient::GetManagedDiskPageRangesDiff`.
```

Comment:
Do we still want this change for the upcoming release, or can we close this PR, ship storage, and revisit the decision in the next release?

Reply:
It's not decided yet.